x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseriesexplorer_controls/timeseriesexplorer_controls.tsx

```diff
@@ -166,7 +170,7 @@ export const TimeSeriesExplorerControls: FC<Props> = ({
         path,
       });
     },
-    [embeddable, selectedJobId, selectedDetectorIndex, selectedEntities]
+    [embeddable, selectedJobId, selectedDetectorIndex, selectedEntities, forecastId]
```

Comment:
If you have more than one SMV embeddable in a dashboard or case, the checkbox for toggling the display of the forecast will always hide / show it from the first panel containing a forecast.

Reply:
Fixed in 71df48b482518e40f776ce667d23a0c2ac65d48c